source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Azure Purview
+date: 2021-03-22 19:38:02
+tags:
+---
+
+If you don't know what kind of data you have and where it resides, it's difficult to use it. Azure Purview can help with that! 
+
+Azure Purview is a unified data governance service that helps you scan and map all your data no matter where the data is stored! tThis data could be on-premise, multi-cloud, or SaaS data.
+
+Azure Purview is an update to the existing Azure Data Catalog Gen 2 service. Azure Data catalog Gen 2 focused more on managing metadata instead of real data governance and was capable of mapping, searching for, and tagging data within data sources but wasn't able to classify its data.This made it hard to be compliant to different laws pertaining data protection, like GDPR. 
```

Comment:
Azure Data catalog Gen 2 >> Azure Data Catalog Gen 2
data.This >> data. This (whitespace)
This made it hard to be compliant to different laws pertaining data protection, like GDPR. >> This made it difficult to comply to various data protection laws, such as GDPR.